### PR TITLE
JBPM-8936 ConcurrentModificationException when retrieving server temp…

### PR DESCRIPTION
**Pull request by bxf12315**
…late

### CODE REVIEW

**Comment by bxf12315** [comment]
> looks ok to me but I'd like to have @mareknovotny and @MarianMacik (or other QE people) opinion

ok

File: pom.xml [hunk 1]
```diff
@@ -6216,7 +6217,15 @@
         <artifactId>javaparser-core</artifactId>
         <version>${version.com.github.javaparser}</version>
       </dependency>
+      
+      <!-- Used by Test Scenario for Thread sleep -->
+	<dependency>
+    	<groupId>org.awaitility</groupId>
+    	<artifactId>awaitility</artifactId>
+    	<version>${version.org.awaitility}</version>
+	</dependency>
```
Comment (mareknovotny) [inline pom.xml]: please indent properly the elements

**Comment by mareknovotny** [comment]
Is this added dependency related to some other PR(s) in other repositories?

File: pom.xml [hunk 2]
```diff
@@ -6216,7 +6217,15 @@
         <artifactId>javaparser-core</artifactId>
         <version>${version.com.github.javaparser}</version>
       </dependency>
+      
+      <!-- Used by Test Scenario for Thread sleep -->
+	<dependency>
+    	<groupId>org.awaitility</groupId>
+    	<artifactId>awaitility</artifactId>
```
Comment (MarianMacik) [inline pom.xml]: Do we even need to use this library? Why not use just a simple `CountDownLatch`.

Comment (mareknovotny) [reply pom.xml]: Please answer @MarianMacik's question too

**Comment by bxf12315** [comment]
It's not necessary, I will close it.

**Pull request closed without merging**